src/js/modules/wallet/controllers/settings/security/settings.security.ctrl.js

```diff
@@ -0,0 +1,513 @@
+angular.module("blocktrail.wallet")
+    .controller("SettingsSecurityCtrl", SettingsSecurityCtrl);
+
+// TODO Review this part, decrease dependencies, create form settings service and move $http request to service
+function SettingsSecurityCtrl($scope, $http, $rootScope, $q, cryptoJS, sdkService, launchService, activeWallet,
+                            $translate, $timeout, $log, $sce, dialogService, accountSecurityService, $filter,
+                            CONFIG, $modal, formSettingsService, passwordStrengthService, settingsService) {
+
+    var settings = settingsService.getReadOnlySettingsData();
+
+    var listenerEnabled2faToggle;
+    // Final 2fa true/false
+    var isEnabled2fa = false;
+    // Toggle for enable in scope
+    $scope.isEnabled2fa = false;
+
+    // TODO: Show information about email verification
+    $scope.hasEmailChanged = false;
+
+    accountSecurityService.updateSecurityScore().then(function () {
+        $scope.accountSecurityInfo = accountSecurityService.getSecurityScore();
+    });
+
+    $rootScope.pageTitle = 'SETTINGS';
+
+    // this automatically updates an already open modal instead of popping a new one open
+    // TODO remove it after moving modals change password, enable/disable 2FA
+    $scope.alert = dialogService.alertSingleton();
+
+    $scope.isLoading = true;
+
+    $scope.errors = {
+        email: false,
+        name: false
+    };
+
+    // Methods
+    $scope.onChangePassword = onChangePassword;
+
+    formSettingsService.fetchData()
+        .then(initData);
+
+    /**
+     * Init data
+     *
+     * @param data
+     */
+    function initData(data) {
+
```

Comment:
we don't need a blank line at the start of every function (see other places as well)